SimGeneral/TrackingAnalysis/python/trackingParticleNumberOfLayersProducer_cff.py

```diff
@@ -0,0 +1,3 @@
+from SimGeneral.TrackingAnalysis.trackingParticleNumberOfLayersProducer_cfi import *
+from Configuration.StandardSequences.Eras import eras
+eras.fastSim.toModify(trackingParticleNumberOfLayersProducer, simHits=['famosSimHits:TrackerHits'])
```

Comment:
Turns out there wasn't one (I was thinking about the refToPSet one). I added #13326. For now, I think the _cff file with the same name as the _cfi is the best approach.